python/src/spark_rapids_ml/core.py

```diff
@@ -943,6 +952,10 @@ def _transformEvaluate(
         """
         raise NotImplementedError()
 
+    def _combine(self, models: List["_CumlModel"]) -> "_CumlModel":
+        """Combine a list of models into a model"""
```

Comment:
In order not to break the compatibility with pyspark, I'd like to throw an exception when calling transform or some other functions for multiple models scenarios or just hide this function for users.

Reply:
I couldn't figure out a way to make "transformEvaluate as a static method" work. Since finally we still need the specific model to do the inference and evaluation.

Reply:
The model list would be passed into the static method.  But I'm fine with throwing exception in methods that don't support multiple models when called on a composite model is fine too.

Reply:
Good suggestion, changed the _combine into static_method